test/test_interface.idl

```diff
@@ -32,4 +32,13 @@ interface TestInterface {
   short shortMethod(short number);
   double doubleMethod(double number);
   string stringMethod(string string);
+
+  // enum
+  void EnumMethod(TestEnum enumValue);
```

Comment:
nit: Should be enumMethod(...) in IDL.
Also, I think better name is voidMethodTestEnumArg(...)

Reply:
i fixed it.